curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-352-blood-tests.md

```diff
@@ -12,41 +12,55 @@ Each one of the 25 sheep in a flock must be tested for a rare virus, known to af
 
-Since the probability of infection for any specific animal is only 0.02, the first test (on the pooled samples) for each group will be: Negative (and no more tests needed) with probability 0.985 = 0.9039207968. Positive (5 additional tests needed) with probability 1 - 0.9039207968 = 0.0960792032.
+The sheep are split into 5 groups of 5 sheep in each group. For each group, the 5 samples are mixed together and a single test is performed. Then,
 
-Thus, the expected number of tests for each group is 1 + 0.0960792032 × 5 = 1.480396016. Consequently, all 5 groups can be screened using an average of only 1.480396016 × 5 = 7.40198008 tests, which represents a huge saving of more than 70% !
+- If the result is negative, all the sheep in that group are deemed to be virus-free.
+- If the result is positive, 5 additional tests will be performed (a separate test for each animal) to determine the affected individual(s).
 
-Although the scheme we have just described seems to be very efficient, it can still be improved considerably (always assuming that the test is sufficiently sensitive and that there are no adverse effects caused by mixing different samples). E.g.: We may start by running a test on a mixture of all the 25 samples. It can be verified that in about 60.35% of the cases this test will be negative, thus no more tests will be needed. Further testing will only be required for the remaining 39.65% of the cases. If we know that at least one animal in a group of 5 is infected and the first 4 individual tests come out negative, there is no need to run a test on the fifth animal (we know that it must be infected). We can try a different number of groups / different number of animals in each group, adjusting those numbers at each level so that the total expected number of tests will be minimised.
+Since the probability of infection for any specific animal is only 0.02, the first test (on the pooled samples) for each group will be:
+
+- Negative (and no more tests needed) with probability ${0.98}^5 = 0.9039207968$.
+- Positive (5 additional tests needed) with probability $1 - 0.9039207968 = 0.0960792032$.
+
+Thus, the expected number of tests for each group is $1 + 0.0960792032 × 5 = 1.480396016$.
+
+Consequently, all 5 groups can be screened using an average of only $1.480396016 × 5 = \mathbf{7.40198008}$ tests, which represents a huge saving of more than 70%!
+
+Although the scheme we have just described seems to be very efficient, it can still be improved considerably (always assuming that the test is sufficiently sensitive and that there are no adverse effects caused by mixing different samples). E.g.:
+
+- We may start by running a test on a mixture of all the 25 samples. It can be verified that in about 60.35% of the cases this test will be negative, thus no more tests will be needed. Further testing will only be required for the remaining 39.65% of the cases.
```

Comment:
```suggestion
- We may start by running a test on a mixture of all 25 samples. It can be verified that in about 60.35% of the cases this test will be negative, thus You will need no more tests. Further testing will only be required for the remaining 39.65% of the cases.
```